cypress/integration/e2e/article.elements.spec.js

```diff
@@ -49,36 +49,13 @@ describe('Elements', function () {
     describe('WEB', function () {
         it('should render the page as expected', function () {
             cy.viewport('iphone-x');
-            const iphoneXWidth = 375;
             let hasElementTooWide = false;
 
             cy.visit(
                 'Article?url=https://www.theguardian.com/commentisfree/2020/jun/30/conservatives-cowboy-builders-boris-johnson',
             );
 
-            const pageHasXOverflow = (docWidth) => {
-                const scrollbarWidth = 15; // Chrome scrollbar is 15px...
```

Comment:
This test is ensuring that the original issue doesn't return, we should keep this I think, is it flakey?

Reply:
The test was flaky and hard to debug. I kept getting errors: `${$el[0].classList[0]}` saying `classList` doesn't exist. Using a selector like `cy.get('*')` makes it hard to know what element in particular it was causing the issue. Idk if there may be another way to check overflow?

Reply:
Have you got a link to the failures in teamcity? I can't find one with this test...

Reply:
> I am able to replicate it locally

This looks like a local issue though? If it isn't happening in Teamcity (and I just checked it isn't happening on my local) it doesn't seem worth deleting the test straight out, I added it because I saw what it tests for broken twice.

The fact that it's getting to that point, suggests that your local is actually breaking!?

Reply:
Hmm, I also just checked out your branch and reverted the test removal, and it seems to pass for me too